collections/_article/godotcon-2025-xr-android-recap.md

```diff
@@ -0,0 +1,135 @@
+---
+title: "What's New in Android & XR"
+excerpt: "Recap of the Android & XR updates during GodotCon 2025"
+categories: ["events", "progress-report"]
+author: "Fredia Huya-Kouadio"
+image: /storage/blog/covers/godotcon-boston-2025-android-xr.webp
+date: 2025-05-30 18:00:00
+---
+
+# What’s New in XR & Android
+
+## GodotCon 2025 recap
+
+A few weeks ago during [GodotCon Boston 2025](https://conference.godotengine.org/2025/), [David Snopek](https://github.com/dsnopek), [Logan Lang](https://github.com/devloglogan) and I gave the latest updates on the state of Android and XR for the Godot engine.  
+You can see the full presentation slides at [https://bit.ly/godotcon-boston-2025-xr-android](https://bit.ly/godotcon-boston-2025-xr-android).
```

Comment:
is the bit.ly here necessary? was it to avoid sharing a long link or to get analytics from it?

I would also add a sentence that the article will be edited to link to the recording whenever it is available on our YouTube channel.

Reply:
Bit.ly was originally used in the presentation to give something remotely typeable on screen. For the blog post, I think it'd be fine to use a direct link to the slides rather than the bit.ly link